### PR TITLE
Task 8

### PR DESCRIPTION
**Pull request by tatthangcntt**
- Fix bugs webchat (TASK-8)  - Done
+ Shows multiple short messages after pressing show/back buttons multi times 
+ Short messages are not showing the same as ones on the Webchat screen 
- Add load more to chat group detail screen

### CODE REVIEW

File: src/stores/chatStore.ts [hunk 1]
```diff
@@ -113,6 +112,8 @@ class ChatStore {
       uniqBy(messages, 'id'),
       'created',
     )
+
+    console.log({ messages2: this.messagesByThreadId[threadId] })
```
Comment (namnm) [inline src/stores/chatStore.ts:116]: Remove console

File: src/components/ChatListWebchats.tsx [hunk 2]
```diff
@@ -10,13 +10,7 @@ const ListWebchats: FC<{
   <>
     {p.datas.map(
```
Comment (namnm) [inline src/components/ChatListWebchats.tsx:11]: Filter first: `p.datas.filter(c => c.webchat).map(item => ...)`

**Review by namnm — CHANGES_REQUESTED** [review]

File: src/stores/chatStore.ts (hunk 1 above)
Comment (tatthangcntt) [reply src/stores/chatStore.ts:116]: Ok! will remove this! Thanks!

File: src/components/ChatListWebchats.tsx (hunk 2 above)
Comment (tatthangcntt) [reply src/components/ChatListWebchats.tsx:11]: I did try but still got synatx of typescript 

image

So, we can keep current or make a new interface with webchat not _undefined_ 
if create new interface it look like that!
image

what's your idea ?

**Pull request merged**